test/e2e/adminUI/pages/home.js

```diff
@@ -0,0 +1,95 @@
+module.exports = {
+	elements: {
+		dashboardHeader: '.dashboard-heading'
+	},
+	sections: {
+		accessGroup: {
+			selector: '.dashboard-group:nth-child(1)',
+			elements: {
+				subheading: '.dashboard-group__heading[data-section-label="Access"]',
+			},
+			sections: {
+				usersTab: {
+					selector: '.dashboard-group__list[data-list-path="users"]',
+					elements: {
+						label: '.dashboard-group__list[data-list-path="users"] .dashboard-group__list-label',
+						plusIconLink: '.dashboard-group__list[data-list-path="users"] a.dashboard-group__list-create.octicon.octicon-plus',
+						itemCount: '.dashboard-group__list[data-list-path="users"] .dashboard-group__list-count',
+					},
+				},
+			},
+		},
+		fieldsGroup: {
+			selector: '.dashboard-group:nth-child(2)',
```

Comment:
ditto: comment for later:  we should replace positional selectors like this with xpath fixed selectors.